src/api/app/views/webui2/webui/interconnects/_interconnect_button.html.haml

```diff
@@ -1,4 +1,5 @@
-.list-group-item
+- name_class = name.tr('.', '_')
```

Comment:
`.` is not the only character that we allow in project names that is not valid in a jQuery id. Check: https://learn.jquery.com/using-jquery-core/faq/how-do-i-select-an-element-by-an-id-that-has-characters-used-in-css-notation/

It may be a good idea to move this to a `helper`, something like `valid_jquery_id`, as you are calling it twice and it can be useful in other places (if we don't have one already, I remember we had one, but maybe it was in javascript and not in Rails and it is possible we dropped it)

Reply:
The more interesting question is why we do this in the view and not wherever we gather the data?

Reply:
This is to generate the `id`, so I think it belongs to the view helper and not to the controller (if that is what you are suggesting @hennevogel)

Reply:
The only purpose of gathering this data is showing it here right? Why would you gather it wrong and then create a helper in between to *"fix"* it?

Reply:
I used another approach. Now the ids of the interconnect list elements, that are just the name of the projects, that can contain the caracters `+`, `.` and `:`, are used without any substitution. I  added the `valid_jquery_id` helper to create a valid jquery accessor to the interconnect list elements. 

Please @Ana06 and @hennevogel have a look again.